RF_Receiver/RF_Receiver.ino

```diff
@@ -268,6 +268,8 @@ void loop() {
 
 //========================= Pulseauswertung ================================================
 void handleInterrupt() {
+  noInterrupts();
```

Comment:
noInterrupts/interrupts habe ich schon in diversen anderen Sketches gesehen und verhindert, dass ein Interrupt den Code eines Interrupts unterbrechen kann. Ist aber nicht zwingend.